iptables/iptables.go

```diff
@@ -247,7 +233,7 @@ func executeCommand(firewallConfiguration FirewallConfiguration, cmd *exec.Cmd,
 		cmd = exec.Command("nsenter", finalArgs...)
 	}
 
-	log.Printf(":; %s\n", strings.Trim(fmt.Sprintf("%v", cmd.Args), "[]"))
+	log.Debugf("%s", strings.Trim(fmt.Sprintf("%v", cmd.Args), "[]"))
```

Comment:
I think this line will print out the actual `iptables` command used, right? Personal preference for me would be to change the level to `info` so it's easier to see what commands are used. We can still leave the final `iptables-save` output as debug though.

Reply:
I agree.  This is the most important piece of information to log so we should make sure it's printed at the default log level.

Reply:
Ok! Adding in next commit.